apps/pie-docs/src/_11ty/shortcodes/card.js

```diff
@@ -0,0 +1,51 @@
+const pieIconsSvg = require('../filters/pieIconsSvg');
+
+/**
+ * A Card HTML component – takes an array of list items and turns them into a list of cards
+ * @param {string[]} items - An array of card items
+ * @returns {string}
+ */
+// eslint-disable-next-line func-names
+module.exports = function ({ items }) {
+    const iconLink = pieIconsSvg({
+        name: 'link-external',
+        attrs: {
+            'aria-hidden': 'true',
+            height: 21,
+            width: 21,
+        },
+    });
+
+    const buildCard = ({
+        label, href, src, shouldOpenInNewTab = false,
+    }) => {
+        const target = shouldOpenInNewTab ? 'target="_blank"' : '';
+
+        const labelClasses = [
+            'c-card-labelContainer',
+            src && 'c-card-labelContainer--hasImage'
+        ].filter(Boolean).join(' ');
+
+        const labelId = `link-${label.replaceAll(' ', '-').toLowerCase()}`;
+
+        return `<article class="c-card" aria-labelledby=${labelId}>
```

Comment:
the `aria` attribute here causes duplication on audible screenreaders when tabbing. I think there's enough context from the link alone that we can remove this